pages/electronics-library-dev.tsx

```diff
@@ -128,8 +158,39 @@ const Home = ({
           </p>
         </div>
       </div>
+
+      <div className={styles.searchBar}>
+        <label htmlFor="search">Search:</label>
+        <input aria-label='Search Electronic Components' type='search' name='search' onChange={onSearchChange} />
```

Comment:
Oh wow. I don't think I've ever realized you could do type='search'. That's awesome.

Reply:
I didn't know that either, but I saw it in a resource I was referencing for this search bar :)